content/en/docs/setup/install/standalone-operator/index.md

```diff
@@ -171,6 +171,37 @@ $ kubectl logs -f -n istio-operator $(kubectl get pods -n istio-operator -lname=
 Refer to the [`IstioOperator` API](/docs/reference/config/istio.operator.v1alpha1/#IstioOperatorSpec)
 for the complete set of configuration settings.
 
+## Canary Upgrade
+You can use operator to do canary upgrade of Istio control plane, similar to what istioctl does, refer to [upgrade guide]((/docs/setup/upgrade/#canary-upgrades))
+
+For example, if you have already installed Istio following the previous step, check existence of in cluster `IstioOperator` CR:
+
+{{< text bash >}}
+$ kubectl get iop --all-namespaces
+NAMESPACE      NAME                        REVISION   STATUS    AGE
+istio-system   example-istiocontrolplane              HEALTHY   11m
+{{< /text >}}
+
+Then run the following command with corresponding version of istioctl to install Istio of new revision based on the in cluster `IstioOperator` CR:
+{{< text bash >}}
+$ istioctl operator init --revision canary
+{{< /text >}}
+
+After running the command, you will have two control plane deployments and services running side-by-side:
+{{< text bash >}}
+$ kubectl get pods -n istio-system -lapp=istiod
```

Comment:
`-l app=istiod` looks more consistent?

Reply:
ok done

Reply:
Thanks, I think one more left to remove.
https://github.com/istio/istio.io/pull/7791/files#diff-4731d0f9bd893d226b43d43608ad14d0R112